baseframe/templates/baseframe/mui/forms.html.jinja2

```diff
@@ -79,6 +79,20 @@
                 <label>{{ field.label.text }}</label>
               {%- endif %}
             </div>
+          {%- elif field.type == 'ImgeeField' %}
+            <div class="imgee">
+              {{ field | render_field_options(class="field-" + field.id + " " + widget_css_class, tabindex=tabindex, autofocus=autofocus, rows=rows) }}
+              <span class="loading imgee__loader mui--hide" id="imgee-loader-{{field.id}}">&nbsp;</span>
+              <a href="#imgee-{{field.id}}" rel="modal:open" class="mui-btn mui-btn--raised imgee__button">{% trans %}Select or Upload Image{% endtrans %}</a>
+              <div class="modal modal--fullscreen" id="imgee-{{field.id}}">
+                <div class="modal__header">
+                  <a class="modal__close mui--text-dark" href="javascript:void(0);" data-target="close cancel register modal" aria-label="{% trans %}Close{% endtrans %}" rel="modal:close">{{ faicon(icon='times', icon_size='title') }}</a>
+                </div>
+                <div class="modal__body">
+                  {{ field.iframe }}
```

Comment:
This `field.iframe` is patched in by the rendering of the field a few lines up. This is shaky. You should instead use `field(iframe=True)` and process that in the widget's `__call__` to return the iframe.